docs/Integrate GitOps with Secrets Management.md

```diff
@@ -0,0 +1,423 @@
+* [Azure Key Vault](https://docs.openshift.com/container-platform/4.14/nodes/pods/nodes-pods-secrets-store.html#secrets-store-azure_nodes-pods-secrets-store)
+
+## Integration guidance with an example using AWS Secrets Manager
+
+### Prerequisites
+* [Your cluster is installed on AWS and uses AWS Security Token Service (STS)](#configure-your-aws-cluster-to-use-aws-security-token-service-sts).
+* [You have configured AWS Secrets Manager to store the required secrets](https://docs.aws.amazon.com/secretsmanager/latest/userguide/create_secret.html).
+* [You have extracted and prepared the `ccoctl` binary](#obtain-the-ccoctl-tool).
+* You have installed the `jq` CLI tool.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+### Install the SSCSID
+To install the Secrets Store CSI driver:
+1. Install the Secrets Store CSI Driver Operator:
+    
+    a. Log in to the web console.
+    
+    b. Click **Operators** → **OperatorHub**.
+
+    c. Locate the Secrets Store CSI Driver Operator by typing "Secrets Store CSI" in the filter box.
+
+    d. Click the **Secrets Store CSI Driver Operator** button.
+
+    e. On the **Secrets Store CSI Driver Operator** page, click **Install**.
+
+    f. On the Install Operator page, ensure that:
+
+    * **All namespaces on the cluster (default)** is selected.
+
+    * **Installed Namespace** is set to **openshift-cluster-csi-drivers**.
+
+    g. Click **Install**.
+
+    After the installation finishes, the Secrets Store CSI Driver Operator is listed in the **Installed Operators** section of the web console.
+
+2. Create the ClusterCSIDriver instance for the driver (`secrets-store.csi.k8s.io`):
+
+    a. Click **Administration** → **CustomResourceDefinitions** → **ClusterCSIDriver**.
+
+    b. On the **Instances** tab, click **Create ClusterCSIDriver**.
+    Use the following YAML file:
+
+    ```
+    apiVersion: operator.openshift.io/v1
+    kind: ClusterCSIDriver
+    metadata:
+        name: secrets-store.csi.k8s.io
+    spec:
+    managementState: Managed
+    ```
+
+    c. Click **Create**.
+
+### Install GitOps
+1. Install the GitOps Operator
+
+Follow the steps on [Installing OpenShift GitOps](OpenShift%20GitOps%20Usage%20Guide.md#installing-openshift-gitops).
+
+2. Bootstrap a GitOps repository
+
+Follow the steps on [Getting started with GitOps Application Manager (kam)](OpenShift%20GitOps%20Usage%20Guide.md#getting-started-with-gitops-application-manager-kam) and [Day 1 Operations](https://github.com/redhat-developer/kam/tree/master/docs/journey/day1).
+
+
+### Store resources of AWS Secrets Manager in the GitOps repository
+1. Add resources for AWS Secrets Manager provider
+
+    a. In your KAM generated GitOps repository, add `secret-provider-app.yaml` for resources of AWS Secrets Manager to `/config/argocd` directory.
```

Comment:
Replace KAM generated GitOps Repository with GitOps repository

Reply:
+1 to that. We clearly do not plan to add or support anything about kam it in our enterprise documentation. Providing kam-related instructions in the documentation means that we will provide support for it if any issue arises from it. The problem that I see here is about maintenance for the R&D, and the decision must be taken by the product team and the PM (@harrietgrace).